packages/test/test-end-to-end-tests/src/test/summarizerWithLocalChanges.spec.ts

```diff
@@ -256,7 +246,16 @@ describeNoCompat("Summarizer with local data stores", (getTestObjectProvider) =>
 				rootDataObject.containerRuntime,
 			);
 			rootDataObject._root.set("dataStore2", dataObject.handle);
-			const { summarizer } = await createSummarizer(provider, container);
+			const { summarizer } = await createSummarizerFromFactory(
```

Comment:
Aside: Maybe `createSummarizerFromFactory` should use a params object to be self-documenting and more concise in the case of lots of `undefined`s

Reply:
Yeah that sounds good. Easier to change params too without changing a bunch of tests. Not doing in this PR though.